terraform/modules/bucket/main.tf

```diff
@@ -2,14 +2,14 @@ data "aws_caller_identity" "current" {}
 
 module "bucket_key" {
   source      = "../key"
-  name        = "${var.name}-bucket"
+  name        = data.aws_s3_bucket.bucket_access_logs.bucket
```

Comment:
The key is for the bucket created in this module, not the access logs bucket. Patch incoming.

Reply:
Also, I think we may run into issues if we start renaming all of the kms keys in legacy.